tests/EdFi.Common.UnitTests/Security/Pbkdf2HmacSha1SecureHasherTests.cs

```diff
@@ -24,39 +26,36 @@ public class When_computing_a_valid_hash : TestFixtureBase
         {
             private PackedHash _actualResult;
             private string _expectedBytes;
-            private SecureHashRequest _request;
             private Pbkdf2HmacSha1SecureHasher _secureHasher;
+            private int _hashAlgorithm;
+            private byte[] _salt;
 
             protected override void Arrange()
             {
-                _secureHasher = new Pbkdf2HmacSha1SecureHasher(null);
+                _secureHasher = new Pbkdf2HmacSha1SecureHasher();
+
+                _hashAlgorithm = HashHelper.GetSha256Hash(Pbkdf2HmacSha1SecureHasher.ConfigurationAlgorithmName).ToInt32();
 
-                _request = new SecureHashRequest
-                {
-                    Secret = Secret,
-                    HashAlgorithm = HashHelper.GetSha256Hash(Pbkdf2HmacSha1SecureHasher.ConfigurationAlgorithmName).ToInt32(),
-                    Iterations = 10000,
-                    Salt = Encoding.UTF8.GetBytes(Salt)
-                };
+                _salt = Encoding.UTF8.GetBytes(Salt);
 
-                _expectedBytes = "9B0FEB3C38F75E8C65BD6516C162F20095F8C3FB6F2006241C17C4D194CF96BD";
+                _expectedBytes = "68EE9DC27ACD5C328D71D2D3096DE9D2833A11C6F1B5F4D14A70C38EC7AB20AB";
```

Comment:
What did we change that caused the hashed bytes to change?

Reply:
hmm looks like i used 100k instead of 10k for the iterations, reverted back to 10k and the answer is correct now.

Reply:
done